cpp/ql/src/experimental/Security/CWE/CWE-703/FindIncorrectlyUsedExceptions.ql

```diff
@@ -18,12 +17,12 @@ import cpp
 from FunctionCall fc, string msg
 where
   exists(ThrowExpr texp |
-    texp.getEnclosingFunction() = fc.(DestructorCall).getTarget() and
+    texp.getEnclosingFunction() = fc.getTarget() and
     not exists(TryStmt ts |
       texp.getEnclosingStmt().getParentStmt*() = ts.getStmt() and
       not ts.getACatchClause().isEmpty()
     ) and
-    msg = "This distructor contains exeption no wrapped to try..catch blocks"
+    msg = "Exception inside this function requires more handling."
```

Comment:
So this is now looking for a throw statement inside a called function, that is not caught/handled inside that function.  This seems like something that normal correct code would be doing fairly frequently, for example a function that's supposed to access a file might reasonably throw an exception if that file doesn't exist.